### DISCUSSION TITLE
How to launch plan without superuser status

### DISCUSSION THREAD

**Discussion by matteocarcano** [frePPLe/frepple#680] [Q&A]
Hi,
i would ask you how can i launch plan if i have user without superuser status. In the execute page i find only the "group and schedule task" but not the "create a plan" folder. I know that i can create a group task but it's not so easy to use for unskilled user.

Thank you for response

![image](https://github.com/user-attachments/assets/93f88f46-a01a-4b02-a493-851a82775fa2)

**Comment by jdetaeye** [comment]
The permissions on the execution screen are "can generate plans" and "run database operations".

**Comment by matteocarcano** [comment]
i have both of them 
![image](https://github.com/user-attachments/assets/5606916a-051f-4ee6-a45e-c3b69ebf5c9e)

**Comment by jdetaeye** [comment]
Hmmmm.  
Just checked it out, and there is a bug indeed in the permission names we use.
